app/serializers/book_serializer.rb

```diff
@@ -0,0 +1,17 @@
+class BookSerializer < ActiveModel::Serializer
+  attributes :id, :title, :author, :genre, :image, :publisher, :year, :available, :actual_rent
+
+  def available
+    return true if object.actual_rent.blank?
+
+    # rubocop:disable Rails/Date
+    !Date.today.between?(object.actual_rent.from, object.actual_rent.to)
+    # rubocop:enable Rails/Date
```

Comment:
Same thing here, the cop deems the `Date.today` method to be unsafe and suggests using `Time.zone.today` instead to take your timezone into account when outputting the date. Try changing your `Date.today` method to fix this cop to avoid disabling too much cops